source/plugins/go/src/oms.go

```diff
@@ -1126,24 +1123,67 @@ func processIncludes(kubernetesMetadataMap map[string]interface{}, includesList
 	includedMetadata := make(map[string]interface{})
 	for _, include := range includesList {
 		switch include {
-		case "podUid":
+		case "poduid":
 			if val, ok := kubernetesMetadataMap["pod_id"]; ok {
 				includedMetadata["podUid"] = val
 			}
-		case "podLabels":
+		case "podlabels":
 			if val, ok := kubernetesMetadataMap["labels"]; ok {
 				includedMetadata["podLabels"] = val
 			}
-		case "podAnnotations":
+		case "podannotations":
 			if val, ok := kubernetesMetadataMap["annotations"]; ok {
-				includedMetadata["podAnnotations"] = val
+				if annotationsMap, ok := val.(map[string]interface{}); ok {
+					filteredAnnotations := make(map[string]interface{})
+					for key, annotationValue := range annotationsMap {
+						if !strings.Contains(key, "kubernetes.io/config") {
+							filteredAnnotations[key] = annotationValue
+						}
+					}
+					includedMetadata["podAnnotations"] = filteredAnnotations
+				}
 			}
 		case "image":
-			if hash, ok := kubernetesMetadataMap["container_hash"]; ok {
-				includedMetadata["image_hash"] = hash
+			// Process Image Hash
+			if hash, ok := kubernetesMetadataMap["container_hash"].(string); ok {
+				if atLocation := strings.Index(hash, "@"); atLocation != -1 {
+					includedMetadata["imageID"] = hash[atLocation+1:]
+				}
 			}
-			if image, ok := kubernetesMetadataMap["container_image"]; ok {
-				includedMetadata["image"] = image
+
+			// Process Image Name
+			if image, ok := kubernetesMetadataMap["container_image"].(string); ok {
+				slashLocation := strings.Index(image, "/")
+				colonLocation := strings.Index(image, ":")
+				atLocation := strings.Index(image, "@")
+				if atLocation != -1 {
+					// Exclude the digest part for repository/image/tag parsing
+					image = image[:atLocation]
+				}
+				if colonLocation != -1 {
+					// Image with tag
+					if slashLocation != -1 && slashLocation < colonLocation {
+						// repository/image:tag
+						includedMetadata["repository"] = image[:slashLocation]
```

Comment:
lets rename from "repository" to "imageRepo"

Reply:
changed